app/views/search/_user_list_element.html.erb

```diff
@@ -0,0 +1,19 @@
+<li class="uk-panel-box uk-grid uk-grid-collapse dark user-search user-result hover uk-padding-remove" onclick="window.location=''">
+  <div class="thumbnail">
+    <%= image_tag photo_path(user.screen_name), class: 'uk-thumbnail' %>
+  </div>
+  <div class="uk-padding-small next-to-thumbnail">
+    <div>
+      <%= user.link_with_compact_name class: 'uk-text-large' %>
+    </div>
+    <% Array(attributes).each do |attribute| %>
+      <div class="uk-text-break">
+        <%= user.send(attribute) %>
+        <div>
+    <% end %>
+    </div>
+    <% selectable = false unless defined?(selectable) %>
+    <% if selectable %>
+      <%= content_tag(:a, 'Kiválaszt', type: :button, class: 'uk-button uk-button-primary', onClick: "selectUser(#{user.id},'#{user.compact_name}')") %>
+    <% end %>
+  </li>
```

Comment:
I'd rather not create this abstraction layer for this partial. Not optimal to duplicate code, but in this case there are slightly different expected behaviors.

For example if you take a look at this code, clicking on the `li` element in the new groups page will reload the page because of the `onclick` handler, not to mention it breaks the searching function. I'd recommend to revert the change in the `app/views/search/_suggest.html.erb` and simplify this partial view.

Also I strongly recommend avoid using `send` method if it's possible. Ruby has a great metaprogramming features, but it's better to use them in a modest way.

If you take a look at it it is much easier to understand this snippet.
```suggestion
<li class="uk-panel-box uk-grid uk-grid-collapse dark user-search user-result hover uk-padding-remove" onclick="selectUser(<%= user.id %>, '<%= user.compact_name %>')">
  <div class="thumbnail">
    <%= image_tag photo_path(user.screen_name), class: 'uk-thumbnail' %>
  </div>
  <div class="uk-padding-small next-to-thumbnail">
    <div>
      <%= user.link_with_compact_name class: 'uk-text-large' %>
    </div>
    <div class="uk-text-break">
      <%= user.room %>
    <div>
    <div class="uk-text-break">
      <%= user.email %>
    </div>
  </div>
</li>
```

Reply:
Agree, this abstraction doesn't feel right.
In my browser selecting a user doesn't reload the page, I used  `event.stopPropagation()` to prevent it.
Could not reproduce the issue with the search function. Where does it break? I'm able to search from the new group page and from /profiles/me page just fine.
l will remove send, and create a new partial for this.

Reply:
Yes, but the `event.stopPropagation()` applies only for the button, but if you click the li element it will reload it, because of this `onclick="window.location=''"`.

For the search, you miss this snippet `onclick="window.location='<%= profile_path(user.screen_name) %>'"`, so when you click on the search result for the user it does nothing.

I think this partial is fine with the suggestion and it's enough to revert the change in the suggest one.